sentry.client.config.ts

```diff
@@ -13,27 +13,26 @@ Sentry.init({
   dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: ["development"].includes(process.env.NODE_ENV) ? 1.0 : 0.1,
+  tracesSampleRate: ["local"].includes(getEnvironment()) ? 1.0 : 0.1,
 
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
-  debug: false,
+  // FIXME set this to `false` after concluding testing, @see MNTOR-2643
+  debug: true,
```

Comment:
Wouldn't something like this be OK to leave in indefinitely?

```suggestion
  debug: getEnvironment() !== "production",
```